src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1698,6 +1698,7 @@ def do_blackbox_fuzzing(self, fuzzer, fuzzer_directory, job_type):
 
   def run(self):
     """Run the fuzzing session."""
+    start_time = time.time()
```

Comment:
LGTM, with one note on this now including fuzzer setup time, I wonder how much time this takes and whether it's more accurate to move the start time to line 1775 just before `do_engine_fuzzing` and `do_blackbox_fuzzing` are called.

Reply:
You could also add a comment before the `FUZZING_SESSION_DURATION`definition to state that this includes fuzzer setup and data bundles update times.

Reply:
I think it is worth measuring the setup time as well, otherwise we would be measuring fuzzing time (which is being tracked in #4356 ) + the time to upload metadata to datastore, which would be dominated by the fuzzing time, and probably make the metric redundant.

Adding the comment as requested